### DISCUSSION TITLE
What novel begins with this phrase?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#879] [Galaxy Trivia] [answered]
*"There was no possibility of taking a walk that day"*

**Comment by SANKARAMDAS** [comment]
Charlotte Brontë’s novel Jane Eyre - as illustrated by Paula Rego

https://www.bl.uk/learning/timeline/item109522.html#:~:text=There%20was%20no%20possibility%20of,%3B%20but%20since%20dinner%20(Mrs.

**Comment by peterbaccaro** [comment, marked as answer]
The novel that begins with the phrase "There was no possibility of taking a walk that day" is **"Jane Eyre" by Charlotte Brontë**. This classic novel, published in 1847, follows the story of Jane Eyre, an orphaned governess, as she navigates her life and relationships. The opening line sets a sombre tone and introduces the reader to Jane's constrained circumstances at the beginning of the story.
